apps/trading/components/navbar/navbar.tsx

```diff
@@ -59,7 +58,11 @@ interface NavLinkProps {
 
 const NavLink = ({ name, path, testId = name }: NavLinkProps) => {
   const router = useRouter();
-  const isActive = router.asPath === path;
+  const [isActive, setIsActive] = useState(false);
```

Comment:
Fixing hydration error. By calling `setIsActive` in `useEffect` a render is triggered after hydrating. https://nextjs.org/docs/messages/react-hydration-error